services/QuillLMS/client/app/bundles/Teacher/components/blog_posts/blog_post_index.jsx

```diff
@@ -21,7 +21,7 @@ export default class extends React.Component {
   pageTitle() {
     const { role } = this.props
     if (window.location.pathname.includes('topic')) {
-      const topicTitle = window.location.pathname.split('/')[3].split('-').map(topic => topic.charAt(0).toUpperCase() + topic.slice(1)).join(' ')
+      const topicTitle = window.location.pathname.split('/')[3].split('-').map(topic => topic.charAt(0).toUpperCase() + topic.slice(1)).join(' ').replace(/%27/g, "'")
```

Comment:
It feels like we really want the Ruby controller to pass this in since we know it's in the `TOPICS` constant there.  That seems cleaner than reverse-engineering our own values via convention.  But I'm not sure how hard that is given our current toolset.